### ISSUE TITLE
RFC: UI improvements

### ISSUE THREAD

**Issue by xl0** [mckaywrigley/chatbot-ui#1781]
I did a slight rework, which includes:
- Use all the horizontal space.
- User/bit icon on the side of the message - saves valuable vertical space.
- Move the side panel collapse button out of the way.
- Honor multi-line user input.

@mckaywrigley How do you feel about it? I think the width could be made an option, and the other 2 changes applied as is.
![Peek 2024-06-09 12-37](https://github.com/mckaywrigley/chatbot-ui/assets/439415/0519d07c-3160-4543-9192-0d9740fcaba5)

https://github.com/mckaywrigley/chatbot-ui/compare/main...xl0:chatbot-ui:compact-layout

**Comment by Bortus-AI** [comment]
Yes def need this. I just merged it with my instance and it works great. Thanks for adding this
